### PR TITLE
Create Iraqueer.org.xml

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

**Comment by pipboy96** [comment]
There are also `ar-iraqueer.org` and `ku-iraqueer.org`. Please also try `mail` sybdomain.

**Comment by pipboy96** [comment]
@Bisaloo Can you make rulesets for Arabic and Kurdish versions of the website?

**Comment by Bisaloo** [comment]
I've added everything in a single ruleset as the number of subdomains is really limited but I can split them if necessary.

File: src/chrome/content/rules/Iraqueer.org.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!--
+	Time out:
+		mail.iraqueer.org
+		mail.ar-iraqueer.org
+
```
Comment (pipboy96) [inline src/chrome/content/rules/Iraqueer.org.xml]: `mail.ku-iraqueer.org`?

Comment (Bisaloo) [reply src/chrome/content/rules/Iraqueer.org.xml]: Seems broken over both http and https

**Pull request merged**